common/dependabot-common.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "aws-sdk-ecr", "~> 1.5"
   spec.add_dependency "bundler", ">= 1.16", "< 3.0.0"
   spec.add_dependency "commonmarker", ">= 0.20.1", "< 0.24.0"
-  spec.add_dependency "docker_registry2", "~> 1.13"
+  spec.add_dependency "docker_registry2", "~> 1.14"
```

Comment:
I just bumped this dep manually in:
* https://github.com/dependabot/dependabot-core/pull/6885

Ideally :dependabot: would open PR's notifying us when this dep bumps so that we don't have to manually track when they have releases... I don't remember enough about the bundler ecosystem to remember why that isn't happening here...

Reply:
Yeah, I think it's because of our "weird" setup (library in common, application in updater, acting somewhat as a monorepo).

Reply:
We'd need to run dependabot against the common gemspec? https://github.com/dependabot/dependabot-core/blob/main/common/dependabot-common.gemspec

Reply:
I think I changed it from common to updater a while ago when I "unified dependencies", because otherwise we were getting duplicated dependencies for RuboCop and others.

Maybe if we added configuration to also bump indirect dependencies to updater/ maybe that could work? 🤔

Reply:
In any case, this release happened 10h ago, so I think we were faster than the bot this time :)